l2rpn_baselines/DoubleDuelingDQN/inspect_action_space.py

```diff
@@ -32,11 +32,12 @@ def default(self, obj):
             return bool(obj)
         else:
             return super(NpEncoder, self).default(obj)
-def cli():
-    parser = argparse.ArgumentParser(description="Action space inspector")
-    parser.add_argument("--path_data", required=True,
-                        help="Path to the dataset root directory")
-    return parser.parse_args()
+# def cli():
```

Comment:
Roll back commented routine, otherwise the script isn't functional

Reply:
ok